examples/sourceMediaIdentifier.xml

```diff
@@ -0,0 +1,14 @@
+<tt xmlns="http://www.w3.org/ns/ttml" 
+    xmlns:ttp="http://www.w3.org/ns/ttml#parameter"
+    xmlns:daptm="http://www.w3.org/ns/ttml/profile/dapt#metadata"
+    xmlns:ebuttm="http://www.w3.org/ns/ttml/profile/dapt#metadata"
```

Comment:
This isn't the right namespace?

Reply:
Good catch, fixed to `urn:ebu:tt:metadata`